LICENSE.md

```diff
@@ -1,31 +1,20 @@
 # Heim Licensing
 
-All software in this repository is open source. Most components are offered
-under a permissive MIT license, but a few are offered under the Affero GPL.
-Files other than source code are offered under the CC-BY 4.0 license.
+All software in this repository is open source. Source code is offered
+under the permissive MIT license. Other files, such as documentation,
+art, and graphics, are offered under the CC-BY 4.0 license.
 
 Please pay careful attention to the license that applies if you reuse or modify
 any of these components.
 
-## aws, backend, cluster, and heimctl
-
-Source code under the `aws`, `backend`, `cluster`, and `heimctl` directories is licensed
-under the terms of the
-[GNU Affero General Public License (Version 3)](https://www.gnu.org/licenses/agpl-3.0.html).
-Copies of this license are available at the root of the repository,
-(`LICENSE.agpl`), as well as in a file named `LICENSE` in each of the
-aforementioned directories.
-
 ## art, doc, and client/res
 
 Files under the `art`, `doc`, and `client/res` directories are offered under
 the terms of the [CC BY 4.0](http://creativecommons.org/licenses/by/4.0/)
 license.  You'll find a copy in the `LICENSE` file in each of these
 directories.
 
-## client and all else
+## aws, backend, client, cluster, heimctl, and all else
```

Comment:
Given that there is no differentiation from the ‘backend’ now, and that is redundant with ‘all else’, one might consider dropping the explicit list.
The directories *not* named yet (and not starting with underscores) are `ci`, `containers`, `heimlich`, `proto`, and `templates` (the last three including source files that *could* have to do with the backend as well :thinking:).